src/interface/src/app/plan/create-scenarios/create-scenarios.component.ts

```diff
@@ -178,9 +169,15 @@ export class CreateScenariosComponent implements OnInit, OnDestroy {
     });
   }
 
-  ngOnDestroy(): void {
-    this.destroy$.next();
-    this.destroy$.complete();
+  pollForChanges() {
+    interval(POLLING_INTERVAL)
+      .pipe(untilDestroyed(this))
+      .subscribe(() => {
+        // only poll when scenario is pending
+        if (this.scenarioState === 'pending') {
+          this.loadConfig();
```

Comment:
We may want to make a new function for this  rather than add loading results into this one— the config values that loadConfig loads in (name, estimated cost, etc) will always be the same so we'd just be loading in the same values every time we check for results